client/web/src/enterprise/batches/detail/BulkOperationsNotifications.tsx

```diff
@@ -0,0 +1,76 @@
+import * as H from 'history'
+import React from 'react'
+
+import { Link } from '@sourcegraph/shared/src/components/Link'
+import { BulkOperationState } from '@sourcegraph/shared/src/graphql-operations'
+import { pluralize } from '@sourcegraph/shared/src/util/strings'
+
+import { DismissibleAlert, isAlertDismissed } from '../../../components/DismissibleAlert'
+import { ActiveBulkOperationsConnectionFields } from '../../../graphql-operations'
+
+export interface BulkOperationsNotificationsProps {
+    location: H.Location
+    bulkOperations: ActiveBulkOperationsConnectionFields
+}
+
+export const BulkOperationsNotifications: React.FunctionComponent<BulkOperationsNotificationsProps> = ({
```

Comment:
Nit, but for consistent language maybe:

```suggestion
export const BulkOperationsAlerts: React.FunctionComponent<BulkOperationsAlertsProps> = ({
```